docs/source/executing-operations/subscription-support.mdx

```diff
@@ -130,17 +130,23 @@ subscription:
       subgraphs: # Overrides subscription settings for individual subgraphs
         reviews: # Overrides settings for the 'reviews' subgraph
           path: /ws # Overrides '/subscriptions' defined above
-          protocol: graphql_transport_ws # The WebSocket-based protocol to use for subscription communication (Default: graphql_ws)
+          protocol: graphql_transport_ws # The WebSocket-based subprotocol to use for subscription communication (Default: graphql_ws)
 ```
 
-This example enables subscriptions in **passthrough mode**, which uses long-lived WebSocket connections. The following WebSocket subprotocols are supported:
+This example enables subscriptions in **passthrough mode**, which uses long-lived WebSocket connections.
 
-- [`graphql_ws`](https://github.com/enisdenjo/graphql-ws)
-- [`subscriptions-transport-ws`](https://github.com/apollographql/subscriptions-transport-ws) (⚠️ unmaintained)
+The router supports the following WebSocket subprotocols, specified via the `protocol` option:
```

Comment:
These two protocols are _so_ difficult to not mix up, would definitely appreciate a double-check here